Robot2018/src/org/usfirst/frc/team199/Robot2018/autonomous/AutoUtils.java

```diff
@@ -0,0 +1,216 @@
+package org.usfirst.frc.team199.Robot2018.autonomous;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class AutoUtils {
```

Comment:
This code is buggy (I found 2 bugs by inspection) and needs tests. Alternatively, use XML or JSON so you don't need to worry about parsing bugs.